packages/actor-context-preprocess-statistic-link-discovery/lib/ActorContextPreprocessStatisticLinkDiscovery.ts

```diff
@@ -0,0 +1,29 @@
+import type { StatisticsHolder } from '@comunica/actor-context-preprocess-set-defaults';
+import type { IActionContextPreprocess, IActorContextPreprocessOutput, IActorContextPreprocessArgs }
+  from '@comunica/bus-context-preprocess';
+import { ActorContextPreprocess } from '@comunica/bus-context-preprocess';
+import { KeysInitQuery, KeysTrackableStatistics } from '@comunica/context-entries';
+import type { IActorTest } from '@comunica/core';
+import { StatisticLinkDiscovery } from './StatisticLinkDiscovery';
+
+/**
+ * A comunica Statistic Link Discovery Context Preprocess Actor.
+ */
+export class ActorContextPreprocessStatisticLinkDiscovery extends ActorContextPreprocess {
+  public constructor(args: IActorContextPreprocessArgs) {
+    super(args);
+  }
+
+  public async test(_action: IActionContextPreprocess): Promise<IActorTest> {
+    return true;
+  }
+
+  public async run(action: IActionContextPreprocess): Promise<IActorContextPreprocessOutput> {
+    const statisticsHolder: StatisticsHolder = action.context.get(KeysInitQuery.statistics)!;
+
+    const statisticLinkDiscovery: StatisticLinkDiscovery = new StatisticLinkDiscovery();
+    statisticsHolder.set(KeysTrackableStatistics.discoveredLinks, statisticLinkDiscovery);
```

Comment:
Let's inline here as well, to reduce the LOC.
The blank lines can then also go, since they don't add much.
Blank lines are mainly useful between complex code blocks. But if a method is this simple, there's no need IMO.